web/src/server/utils/withMiddlewares.ts

```diff
@@ -0,0 +1,46 @@
+import { isPrismaException } from "@/src/utils/exceptions";
+import { cors, runMiddleware } from "@/src/features/public-api/server/cors";
+import { type NextApiRequest, type NextApiResponse } from "next";
+import { z } from "zod";
+import { BaseError } from "@langfuse/shared";
+
+export function withMiddlewares(
```

Comment:
Thanks for introducing that! Can we rename this to soemthing like `wityhGlobalErrorMiddleware`? WDYT?